jupyter-extension/jupyterlab_pachyderm/tests/test_integrations.py

```diff
@@ -94,8 +94,16 @@ async def test_mount(pachyderm_resources, http_client: AsyncClient):
     assert len(resp["mounted"]) == 3
     mounted_names = [mount["name"] for mount in resp["mounted"]]
     assert len(resp["unmounted"]) == 2
-    assert len(resp["unmounted"][0]["branches"]) == 1
-    assert len(resp["unmounted"][1]["branches"]) == 2
+    for repo in r.json()["unmounted"]:
+        if repo["repo"] == repos[1]:
+            print(repo)
```

Comment:
Probably want to delete these print statements?

Reply:
whoops, good catch haha